test/mocks/redux/redux_store_mocks.dart

```diff
@@ -2,26 +2,38 @@ import 'dart:async';
 
 import 'package:crowdleague/actions/redux_action.dart';
```

Comment:
We now have just one mock/fake store - I think Fake is the appropriate name... from a random article I found: 
> Fakes. These objects actually have a complete working implementation in them. But the implementation provided in them is some kind of shortcut which helps us in our task of unit testing, and this shortcut renders it incapable in production. A great example of this is the in-memory database object which we can use just for our testing purposes, while we use the real database object in production.

Reply:
The FakeStore holds the app state and allows state to be updated, such that widgets under test will be rebuilt with the new state.  The FakeStore also keeps track of actions that have been dispatched by the widgets under test.